### PR TITLE
Add tutorial and script for GKE

### PR DESCRIPTION
**Pull request by askore**

### CODE REVIEW

File: tutorials/README.md [hunk 1]
```diff
@@ -182,19 +189,100 @@ For example:
...
+
+```
+model_name=critterblock
+gcp_project_id=test-project-12345
+docker_image_tag=v5
+cluster_name=test-cluster
+gcp_region=us-central1
+```
+
+Run the `deploy_to_eks.sh` script to create an artifact repository, push your image to it, and configure kubectl to connect to your cluster:
```
Comment (vanessahlyan) [inline tutorials/README.md:257]: deploy_to_eks or deploy_to_gke?

File: tutorials/deploy_to_gke.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+set -eu
+
+MODEL_NAME=$1
+#SANDBOX_DEV_PROFILE=$2
+PROJECT_ID=$2
+DOCKER_IMAGE_TAG=$3
+#AWS_REGISTRY_URL=$4
```
Comment (vanessahlyan) [inline tutorials/deploy_to_gke.sh:8]: need to make sure we don't mention AWS here and in other places of this script

File: tutorials/README.md [hunk 3]
```diff
@@ -182,107 +188,139 @@ For example:
...
-## Express Acceleration Mode and Full Acceleration Mode
+And run inference with http:
 
-The following table explains the difference between the two modes
+```
+python run.py --triton --protocol http --port 8080
+```
 
 
-| Name | Express Acceleration Mode | Full Acceleration Mode |
```
Comment (nibalizer) [inline tutorials/README.md:280]: Where did my table go?

Comment (askore) [reply tutorials/README.md:280]: Looks like a bad merge. Trying to figure it out.

**Comment by askore** [comment]
Not on this branch, but I haven't completely ironed out the process for releasing new versions of the chart.

**Pull request merged**